optuna_dashboard/_serializer.py

```diff
@@ -108,6 +108,7 @@ def serialize_study_summary(summary: StudySummary) -> dict[str, Any]:
         "study_name": summary.study_name,
         "directions": [d.name.lower() for d in summary.directions],
         "user_attrs": serialize_attrs(summary.user_attrs),
+        "is_preferential": summary.system_attrs.get(_SYSTEM_ATTR_PREFERENTIAL_STUDY, False),
```

Comment:
`system_attrs` property was actually deprecated.
https://github.com/optuna/optuna/blob/247093640034f7313b4ee3dce18903105bc2a7fd/optuna/study/_study_summary.py#L114-L123

Could you please access to `_system_attrs` by using `getattr` function?

```suggestion
        "is_preferential": getattr(summary, "_system_attrs", {}).get(_SYSTEM_ATTR_PREFERENTIAL_STUDY, False),
```